build.gradle.kts

```diff
@@ -14,10 +14,31 @@ application {
 repositories {
     jcenter()
     mavenCentral()
+
+    // astminer repo
+    maven(url = "https://dl.bintray.com/egor-bogomolov/astminer/")
 }
 
 dependencies {
     testImplementation(kotlin("test-junit"))
+
+    implementation("org.jetbrains.kotlin", "kotlin-reflect", "1.4.10")
+    // astminer
+    implementation("io.github.vovak.astminer", "astminer", "0.6")
+    // dependency astminer
+    implementation("com.github.ajalt.clikt", "clikt", "3.0.1")
```

Comment:
This is a library for CLI arguments, it's not related to astminer.

Reply:
fixed